src/main/java/pl/envelo/moovelo/controller/event/EventController.java

```diff
@@ -237,7 +239,7 @@ public ResponseEntity<List<EventSurveyDto>> getEventSurveysByEventId(@PathVariab
                             authorizationService.isLoggedUserEventOwner(eventId)) {
                         return EventSurveyMapper.mapEventSurveyToEventSurveyDto(surveyDto);
                     } else {
-                        BasicUser basicUser = (BasicUser) user;
+                        BasicUser basicUser = (BasicUser) authorizationService.getLoggedUser();
```

Comment:
zrobiłam metodę authorization.getLoggedBasicUser() - można ją tu wrzucić. może dodaj todo i jak się zmerguje moje zmiany, to wtedy się to przy okazji zrobi